### PR TITLE
SPE-788: Negotiation helper + authority graph review hardening

### PR DESCRIPTION
**Pull request by JamesJedi420**
## Summary

- Linear: [SPE-788](https://linear.app/spectranoir/issue/SPE-788/authority-relationship-graph-and-politics-layer)
- **Scope:** Slice 1 landed on `main` via #2155; **this PR** adds slice 2 (negotiation helper) plus **slice 1 review hardening** (resolver + tests still in diff because hardening landed here before/after merge).

> **Note for reviewers:** `authorityGraph.ts` is not net-new slice 1 — it is the merged #2155 baseline plus deterministic fixes (proxy pairing, subordination direction, pair-scoped contradictions, magnitude clamp, token collector). The negotiation module is the primary new surface area.

## Files changed

| File | Role |
|------|------|
| `src/domain/authorityGraph.ts` | Slice 1 resolver **delta** (review hardening on #2155 baseline) |
| `src/test/authorityGraph.test.ts` | Slice 1 regressions + hardening proofs (4b, 15–16b, 17–19) |
| `src/domain/authorityNegotiation.ts` | **Slice 2** bargaining outcomes over graph consequences |
| `src/test/authorityNegotiation.test.ts` | Slice 2 + hardening regressions |

## Validation

- `npm run test:run -- src/test/authorityGraph.test.ts` — 24 passed
- `npm run test:run -- src/test/authorityNegotiation.test.ts` — 20 passed
- `npm run lint` — passed
- `git diff --check` — passed

## Review hardening

- Proxy `representsNodeId` actor/counterparty matching
- Reverse-edge peer directionality (subordination grant vs modify)
- Contradiction flags scoped to actor/counterparty pair; emit loop uses `edgeAppliesToQuery`
- `includeContradictedClaims` honored for status/confidence
- Merged magnitudes clamped to -100..100
- Token collector includes linked ids, `representsNodeId`, `recorderId`
- Negotiation: channel-scoped pair hints (`pressureChannels`), agenda dilution without negative grant, baseline `edgeIds` scoped per channel
- Unrelated graph contradictions do not force `grudging_alignment` (negotiation test 18)

## Gap / edge-case / boundary audit

- Represented-actor proxy queries resolve; subordination direction preserved
- Unrelated contradictions do not leak into other pair negotiations
- Unknown actor/counterparty remains safe (`procedural_block`, empty baseline)
- Deterministic ordering; graph/request not mutated
- No GameState, routing, UI, history, ethics, or `factions.ts` changes

## Explicitly out of scope

- `authorityGraphHistory` / graph mutation over campaign weeks
- Expanding slice-1 edge-kind resolver coverage beyond review fixes
- GameState persistence, `missionIntakeRouting`, `advanceWeek`, UI
- SPE-1047 / SPE-1096 / SPE-1097 / SPE-2088 / SPE-2089 / secrecy-media runtime
- `factions.ts` refactor; source-literal faction names or lore

## Test plan

- [x] Authority graph unit suite
- [x] Authority negotiation unit suite
- [x] ESLint

### CODE REVIEW

**Comment by JamesJedi420** [comment]
## Review hardening — fixed in 9c8f6a3

Thanks for the review threads. This commit addresses all reported merge blockers and cleanup items without expanding scope.

### authorityGraph.ts
1. **Proxy actor matching** — edgeInvolvesNode includes 
epresentsNodeId for proxy_representation; pair queries require both endpoints. Test 4b.
2. **Peer directionality** — 
esolvePeerNodeId returns the opposite endpoint; subordination grant only when subordinate queries superior. Test 15.
3. **Contradiction scoping** — collectContradictionEdgeIdsForPair limits flags to the queried pair; removed graph-global mixed-channel contradiction_flag injection. Test 16.
4. **includeContradictedClaims** — contradicted status/confidence edges pass when requested. Test 17.
5. **Magnitude clamp** — merged consequences clamped to -100..100 via clampMagnitude. Test 18.
6. **Token collection** — linked ids, 
epresentsNodeId, 
ecorderId included. Test 19.

### authorityNegotiation.ts + tests
7. **Type imports** — graph fixture types imported from uthorityGraph in tests.
8. **Agenda dilution** — no negative grant without existing aid grant; uses modify when trimming absent grant. Test 16.
9. **Channel-scoped hints** — pair hints respect pressureChannels. Tests 15, 18.
10. **
esult.delayed** — true when baseline, retaliation outcome, or effective/adjustment delay. Test 17.
11. **Outcome coverage** — per-scenario expect(run()).toBe(outcome) assertions.

Validation: 41 targeted tests + full suite 3323 green; lint clean.

File: src/domain/authorityNegotiation.ts [hunk 1]
```diff
@@ -0,0 +1,587 @@
...
+): PairGraphHints {
+  const activeKinds = new Set<AuthorityRelationshipKind>()
+  let maxVolatility = 0
+
+  for (const edge of graph.edges) {
+    if (!edgeInvolvesPair(edge, actorId, counterpartyId) || !edgeIsActive(edge, asOfWeek)) {
+      continue
+    }
+
+    activeKinds.add(edge.kind)
```
[the remark this answers is not in the document]
Comment (JamesJedi420) [reply src/domain/authorityNegotiation.ts:213]: **Disposition (already fixed in 9c8f6a3; proof tests in 58f525a)**

\collectPairGraphHints\ filters with \edgeAppliesToChannel\: edges apply when \pressureChannels\ is absent/empty (channel-general) or includes \
equest.channel\.

Regression test 15 now asserts:
- \hidden_agenda\ scoped to \information_flow\ does **not** yield \genda_dilution\ on \permission\; does yield it on \information_flow\.
- \shared_authority\ scoped to \secrecy\ does **not** yield \grudging_alignment\ on \id\; does yield it on \secrecy\.

Targeted + full Vitest green on 58f525a.

**Comment by JamesJedi420** [comment]
@copilot resolve the merge conflicts in this pull request
